docker/gaffer-kerberos/gaffer-integration-tests-krb/accumulo-int-entrypoint.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+# Copyright 2023 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Wait a minute for Accumulo to be started and working
+sleep 60
+
+# Grant required permissions and auths to Gaffer user for integration tests
+
+PRINCIPAL=accumulo/$(hostname)
+FULL_PRINCIPAL=$PRINCIPAL@GAFFER.DOCKER
+GAFFER_FULL_PRINCIPAL=$1
+
+kinit -k -t /etc/accumulo/conf/accumulo.keytab $FULL_PRINCIPAL
+
+echo "\nGranting permissions for Gaffer integration tests\n"
+
+if echo "$ACCUMULO_VERSION" | grep -q "^2.*$"; then
```

Comment:
Is the regex maybe missing an escaped `.` e.g. "^2\..*`

Reply:
This exists in a few other places too, might be better to put this into a new issue with the other suggestions like this.